src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { capitalize } from 'lodash';
 
 // modeled after VA Profile's Health Benefit AssociatedPersonBio
 
+// const CONTACT_TYPES = [
+//   'Emergency Contact',
+//   'Other emergency contact',
+//   'Primary Next of Kin',
+//   'Other Next of Kin',
+// ];
```

Comment:
Should we remove the CONTACT_TYPES, and maybe add a kind of enum proptype to the contactType here?
something like:
```
contactType: PropTypes.oneOf([ 'Emergency Contact', ...])
or even
contactType: PropTypes.oneOf(Object.keys(DESCRIPTIONS))
```

Just a thought

Reply:
Good call. I'll do that.